benchmark-scripts/benchmark.sh

```diff
@@ -13,6 +13,7 @@ error() {
 show_help() {
         echo "
          usage: $0 
+           --performance_mode the system performance setting
```

Comment:
could you add what options you can have when user set this?
```suggestion
           --performance_mode the system performance setting [powersave | performance]
```